azure-stack/hci/manage/manage-secure-baseline.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Manage baseline security settings on Azure Stack HCI, version 23H2 (preview)
+description: Learn about managing baseline security settings available for new deployments of Azure Stack HCI, version 23H2 (preview).
+author: alkohli
+ms.author: alkohli
+ms.topic: how-to
+ms.service: azure-stack
+ms.subservice: azure-stack-hci
+ms.date: 12/04/2023
+---
+
+# Manage baseline security settings for Azure Stack HCI, version 23H2 (preview)
+
+[!INCLUDE [hci-applies-to-23h2](../../includes/hci-applies-to-23h2.md)]
+
+This article describes how to manage baseline security settings for your Azure Stack HCI cluster and the associated drift control mechanism to ensure that the device starts in a known good state.
+
+[!INCLUDE [important](../../includes/hci-preview.md)]
+
+## Enable, disable, or toggle drift control
+
+To adjust security hardening, we recommend a balanced security posture. Use the initial security baseline, toggle drift control, and modify protected security settings defined during deployment.
+
+### Enable drift control
+
+Use the following steps to enable drift control:
+
+1. Connect to your Azure Stack HCI node.
+1. Run the following cmdlet:
+
+    ```PowerShell
+    Enable-AzsSecurity -FeatureName DriftControl -Scope <Local | Cluster>
+    ```
+
+   - **Local** - Affects the local node only. Can be run from a remote PowerShell session.
+   - **Cluster** - Affects all nodes in the cluster using the orchestrator. Requires user to belong to the deployment authorization group (PREFIX-ECESG) and CredSSP, or an Azure Stack HCI server using a remote desktop protocol (RDP) connection.
+
+### Disable drift control
+
+Use the following steps to disable drift control:
+
+1. Connect to your Azure Stack HCI node.
+1. Run the following cmdlet:
+
+    ```PowerShell
+    Disable-AzsSecurity -FeatureName DriftControl -Scope <Local | Cluster>
+    ```
+
+   - **Local** - Affects the local node only. Can be run from a regular remote PowerShell session.
+   - **Cluster** - Affects all nodes in the cluster using the orchestrator. Requires user to belong to the deployment authorization group (PREFIX-ECESG) and CredSSP, or an Azure Stack HCI server using a remote desktop protocol (RDP) connection.
+
+### Toggle drift control
+
+Use the following steps to toggle drift control:
+
+1. Connect to your Azure Stack HCI node.
+1. Run the following cmdlet using local administrator credentials or deployment user account credentials.
+
+## Configure security during deployment
+
+When deploying your cluster via the Supplemental Package, you can modify drift control settings and other security settings that constitute the security baseline. Changes you make to security settings are also reflected in the *config.json* file that you create using the deployment tool.
```

Comment:
Change: When deploying your cluster via the Supplemental Package

To: When deploying your cluster,

Reply:
Remove this: Changes you make to security settings are also reflected in the *config.json* file that you create using the deployment tool.

Reply:
Done > updated.